includes/weather-functions.php

```diff
@@ -37,9 +37,21 @@ function get_weather( $cache_key ) {
 		}
 
 		if ( $json_url ) {
-			$json = array();
-			$args = array(
-				'timeout' => get_option( 'weather_service_timeout' )
+			// Setup curl request and execute. Log errors if necessary.
+
+			$dt = urlencode( microtime() );
+
+			if ( strpos( $json_url, '?' ) > -1 ) {
+				$json_url .= "&request_time=$dt";
+			} else {
+				$json_url .= "?request_time=$dt";
+			}
+
+			$ch = curl_init();
```

Comment:
`curl_init()` call should be removed since we're using `wp_remote_get()` below.

Reply:
This and the other error were due to some sloppy merging. I was working on outdated code to begin with and never updated this after I get the newest code merged in. All fixed up now!